index.html

```diff
@@ -54,164 +62,153 @@ <h2>LET'S CONNECT</h2>
-        </div>
-      </section>
+                  <div class="card-container-1">
+                    <img src="images/snapshoot-portfolio-3.svg" alt="">
+                    <div class="card-content-1">
+                    <h3>multi-post stories</h3>
+                    <h4><span>Facebook</span>&nbsp;
+                      <img src="images/counter.svg" alt="">&nbsp;
+                      full stack dev&nbsp;
+                      <img src="images/counter.svg" alt="">&nbsp;
+                      2015
+                    </h4>
+                    <p>Exploring the future of media in facebook's first virtual Reality <br>
+                      app a place to discover and enjoy 360 points and videos on <br>
+                      Gear VR.</p>
+                    <ul>
+                      <li>html</li>
+                      <li>Ruby on rails</li>
+                      <li>css</li>
+                      <li>javascript</li>
+                    </ul>
+                    <button>see project</button>
+                    </div>
+                  </div>
+                </article>
+              </section>
+              <section class="about">
+                <div class="about-container">
+                  <div class="about-content">
+                  <h2>About <br>Myself</h2>
+                  <p>Hello I’m a software developer! I can help you build a product ,<br> feature
+                    or website Look through some of my work and experience!<br> If you like what
+                    you see and have a project you need coded, <br> don’t hestiate to contact me.</p>
+                  <p class="connect">let's connect</p>
+                  <ul class="social">
+                    <li><a href="#"><img src="images/twitter.svg" alt=""></a></li>
+                    <li><a href="#"><img src="images/linkedin.svg" alt=""></a></li>
+                    <li><a href="#"><img src="images/medium.svg" alt=""></a></li>
+                    <li><a href="#"><img src="images/github.svg" alt=""></a></li>
+                    <li><a href="#"><img src="images/angellist.svg" alt=""></a></li>
+                  </ul>
+                  <button class="about-btn">get my resume</button>
+                  </div>
+                  <div class="btn-icon-language">
+                  <h3><span>languages</span><img src="images/arr-down.svg" alt=""></h3>
+                  <ul class="stack">
+                    <li><img src="images/js.svg" alt="">&nbsp;&nbsp;&nbsp;<span>JavaScript</span></li>
+                    <li><img src="images/html.svg" alt="">&nbsp;&nbsp;&nbsp;<span>HTML</span></li>
+                    <li><img src="images/css.svg" alt="">&nbsp;&nbsp;&nbsp;<span>CSS</span></li>
+                  </ul>
+                  <h3><span>frameworks</span><img src="images/arr-right.svg" alt=""></h3>
+                  <h3 class="skills"><span>skills</span><img src="images/arr-right.svg" alt=""></h3>
+                  </div>
```

Comment:
- [ ] Please follow the design guidelines for this section as it looks very different from what it ought to be
### Your Design
![image](https://user-images.githubusercontent.com/46839548/168427850-6d6425b7-dced-4fc7-abda-4c0042dac57b.png)
### Figma Design
![image](https://user-images.githubusercontent.com/46839548/168427905-a26ec9d2-cf18-4373-8fff-1b5bfbb20aed.png)